exec/java-exec/src/main/java/org/apache/drill/exec/ExecConstants.java

```diff
@@ -49,6 +49,7 @@
   String USER_RPC_TIMEOUT = "drill.exec.rpc.user.timeout";
   String METRICS_CONTEXT_NAME = "drill.exec.metrics.context";
   String USE_IP_ADDRESS = "drill.exec.rpc.use.ip";
+  String BIT_ADVERTISED_HOST = "drill.exec.rpc.bit.advertised.host";
```

Comment:
Is this really what we want to do? After this change, each host will have its on Drill config file. That means that, on every config change, the admin must:

1. Modify a master file.
2. Use a script to regenerate the per-host files.
3. Push the per-host files to the remote systems.
4. Restart the cluster.

If Drill is deployed on a system such as YARN, it is not possible to have per-host config files.